packages/pf4-component-mapper/src/files/wizard.js

```diff
@@ -89,27 +85,33 @@ const WizardInternal = ({
         {title && <WizardHeader title={title} description={description} onClose={formOptions.onCancel} />}
         <div className="pf-c-wizard__outer-wrap">
           <WizardNav>
-            <FormSpy>
-              {({ values }) => (
+            <FormSpy subscription={{ values: true, valid: true, validating: true }}>
```

Comment:
what about submitting? I think we should block the navigation while submitting as well.

Reply:
the whole navigation? this just disables next steps (on the last step there are no next steps)

Reply:
Is it ok to move in the wizard when the data is in the submission process? I think the navigation in the wizard should be postponed.

Reply:
After submit, the wizard form should go away and be replace by spinner/loading progress according to design specs - there should be no navigation or buttons at all.

Reply:
Yes but we are talking about the time period when the submit is in progress not when it was finished.